Source/Core/Core/HW/CPU.cpp

```diff
@@ -165,6 +211,13 @@ void CPUManager::Run()
       break;
     }
   }
+
+  if (Config::Get(Config::MAIN_TIME_TRACKING))
```

Comment:
Please don't check the config here again for a function-local state, you can't guarantee that the config hasn't changed since the start. `if (timing.joinable())` would be a better check here.